os/fs/smartfs/smartfs_smart.c

```diff
@@ -461,6 +467,9 @@ static int smartfs_close(FAR struct file *filep)
  * Name: smartfs_read
  ****************************************************************************/
 
+#ifdef CONFIG_SMART_FS_SECURE
+unsigned char output[MAXBUFSIZE];
```

Comment:
Maybe static

Reply:
This variable is also used in another file (smartfs_utils.c) .

Reply:
Still you can make "output" variable as "static" as you are passing it as variable to smartfs_encrypt() function.